src/extensions/renderer/base/index.js

```diff
@@ -23,10 +23,12 @@ BRp.init = function( options ){
   r.cy = options.cy;
 
   var ctr = r.container = options.cy.container();
+  var containerWindow = ctr.ownerDocument.defaultView || window;
```

Comment:
Another point of consideration re. this sort of line:  Cytoscape has fairly good support for older environments (even IE).

I wouldn't suggest that you should test this on old browsers, but we ought to make at least a sensible best effort not to break things where some API may not be available.  If there were a reusable `cy.window()` function, for instance, then it would be simple to add null guards in that function -- in case the environment doesn't have `ownerDocument` defined, in case the container isn't defined, etc.